### DISCUSSION TITLE
Issues handling HTML form input with PHP

### DISCUSSION THREAD

**Discussion by RoujinX** [yandeu/five-server#47] [Help / Questions]
It seems I cant handle HTML form input with PHP using FiveServer; am I missing a setting or is this a bug?
The same code works fine in Atom with PHP server package and also in VScode with PHP server extension.
With FiveServer the as soon as I submit my (form) input the server reloads (disconnects and immediately reconnects) and nothing else happens. I've tried to mess around with several settings and nothing seems to work, but all other PHP code seems to work perfectly
![image](https://user-images.githubusercontent.com/20985987/152440032-0b845f83-8878-4f8e-9666-706d480e030b.png)

**Comment by yandeu** [comment]
Five-server renders PHP, but it does not provide a PHP server.
Try this setup:
https://github.com/yandeu/five-server-with-php#readme
